code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -192,11 +201,15 @@ Please contact me on #coderbus IRC. ~Carnie x
 
 /mob/living/carbon/human/update_inv_wear_id()
 	remove_overlay(ID_LAYER)
+
 	if(wear_id)
 		if(client && hud_used && hud_used.hud_shown)
 			wear_id.screen_loc = ui_id
 			client.screen += wear_id
 
+		if(dna && istype(dna.species, /datum/species/pony))

```

Comment:
it involves using some sort of species flag to tell the icon updating to just fuck off.